src/olympia/addons/migrations/__init__.py

```diff
@@ -0,0 +1,43 @@
+from urllib.parse import urlsplit, urlunsplit
+
+# for src\olympia\addons\migrations\0037_auto_20220321_1545.py
+
+CONTRIBUTION_CHANGES = {
+    'buymeacoffee.com': 'www.buymeacoffee.com',
+    'www.donate.mozilla.org':  'donate.mozilla.org',
+    'www.flattr.com': 'flattr.com',
+    'www.github.com': 'github.com',
+    'www.ko-fi.com': 'ko-fi.com',
+    'www.liberapay.com': 'liberapay.com',
+    'id.liberapay.com': 'liberapay.com',
+    'micropayment.de': 'www.micropayment.de',
+    'www.opencollective.com': 'opencollective.com',
+    'patreon.com': 'www.patreon.com',
+    'paypal.com': 'www.paypal.com',
+    'paypal.me': 'www.paypal.com',
+}
+
+BROKEN_URLS = (
+    'https://www.fdfgd@paypal.me',
+    'https://gist.github.com/miry/2a7635beaa7078e0d7af',
+    'https://thezenithpoint.co.uk@donate.mozilla.org',
+    'https://bunny.github.com',
+)
+
+
+def fix_contribitions_url(url):
```

Comment:
btw, this code is in here rather than the migration file itself because I wanted it to have test coverage but importing from a filename that starts with numbers is a giant pain in python.

Reply:
typo in the name of the function `contribitions` vs `contributions` (which you seem to have copy/pasted so at least it doesn't break...)